drizzle-orm/src/planetscale-serverless/driver.ts

```diff
@@ -22,9 +23,26 @@ export type PlanetScaleDatabase<
 > = MySqlDatabase<PlanetscaleQueryResultHKT, PlanetScalePreparedQueryHKT, TSchema>;
 
 export function drizzle<TSchema extends Record<string, unknown> = Record<string, never>>(
-	client: Connection,
+	client: Client,
 	config: DrizzleConfig<TSchema> = {},
 ): PlanetScaleDatabase<TSchema> {
+	if (!(client instanceof Client)) {
```

Comment:
Please use `is()` from `drizzle-orm` package

https://orm.drizzle.team/docs/goodies#compare-objects-types-instanceof-alternative

```ts
if (!is(client, Client)) {
...
```